manifests/cloner/script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ "$1" == "source" ] ; then
+  mkfifo /tmp/clone/socket/pipe; cat /tmp/clone/image/disk.img > /tmp/clone/socket/pipe
```

Comment:
In order to not be kubevirt specific you should use tar:

```
tar cv /tmp/clone/image/ > /tmp/clone/socket/pipe
```

Please add logging at each step.

Reply:
so this means that _disk.img_ is not hard-coded in the cloner, though it is hard-coded in the importer.

Reply:
Yeah.  Better to keep cloning general purpose from the start.